molo/core/api/compat.py

```diff
@@ -0,0 +1,12 @@
+"""
+Current molo implementation depends on wagtail v1.4.5.
```

Comment:
Note: we're currently upgrading molo to `1.8` - we'll have to make sure this is handled.

tip: We can use `try... except...` to handle backwards compatibility